runtime/src/main.rs

```diff
@@ -437,6 +461,49 @@ fn convert_result<T>(
     }
 }
 
+fn extract_entrypoint(config: &ContainerConfig) -> Option<Vec<String>> {
+    let mut entrypoint = config.entrypoint.clone().unwrap_or_default();
+    entrypoint.append(&mut config.cmd.clone().unwrap_or_default());
+    if entrypoint.is_empty() {
+        None
+    } else {
+        Some(entrypoint)
+    }
```

Comment:
cosmetic change suggestion

```suggestion
    config.entrypoint.clone().map(|mut entrypoint| {
        entrypoint.extend(config.cmd.clone().unwrap_or_default());
        entrypoint
    })
```

Reply:
Your suggestion would only work if entrypoint is defined. Docker would run CMD as entrypoint if entrypoint is not defined.

Reply:
Changing `append` to `extend`.

Reply:
You’re right, we we would still need to .or() cmd

Reply:
@mfranciszkiewicz does this look better?
    let entrypoint = config.entrypoint.clone().unwrap_or_default().into_iter()
        .chain(config.cmd.clone().unwrap_or_default())
        .collect::<Vec<_>>();